.github/workflows/acctest-terraform-lint.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
     paths:
       - .github/workflows/acctest-terraform-lint.yml
-      - .go-version
+      - go.sum
```

Comment:
Do we want to do a more widespread application of this change?

Reply:
Yes, #29843